TombEngine/Renderer/RendererSprites.cpp

```diff
@@ -261,16 +263,16 @@ namespace TEN::Renderer
 			return;
 
 		// Draw instanced sprites.
-		bool wasGPUSet = false;
-		for (auto& spriteBucket : _spriteBuckets)
+		bool wasGpuSet = false;
```

Comment:
Not sure if it should be Gpu, because GPU is not a name, it is an abbreviation.

Reply:
Abbreviations longer than two characters are still treated as words by convention. In fact I think even two-letter abbreviations should be treated like regular words too because of some oddities you can get, but that's another matter. :)